src/libbson/src/bson/bson-context-private.h

```diff
@@ -31,10 +31,10 @@ BSON_BEGIN_DECLS
 struct _bson_context_t {
    /* flags are defined in bson_context_flags_t */
    int flags;
-   int32_t seq32;
-   int64_t seq64;
-   uint8_t rand[5];
-   uint16_t pid;
+   uint32_t seq32;
+   uint64_t seq64;
+   uint8_t randomness[5];
```

Comment:
```suggestion
   uint8_t randomness[BSON_CONTEXT_RANDOMNESS_SIZE];
```
Magic number; replace with named constant?